src/views/layouts/_page.njk

```diff
@@ -24,7 +24,7 @@
                 "id": "main-navigation",
                 "classes": "ons-nav--inline ons-nav--light ons-nav--header ons-nav--h-m ons-js-main-nav",
                 "ariaLabel": "Main menu",
-                "currentPath": [pageInfo.url, pageInfo.parent.url],
+                "currentPath": [pageInfo.url, pageInfo.parent.url if pageInfo.parent.url != '/' else ''],
```

Comment:
Is this intended to be a part of this PR?

Reply:
No, I will remove now